.github/workflows/release_pdb.yml

```diff
@@ -0,0 +1,67 @@
+name: Release XSPdb Jobs
+on:
+  push:
+    branches: [pdb]
+  pull_request:
+    branches: [pdb]
+jobs:
+  build-xsdev-image:
```

Comment:
`build-xsdev-image` was a bad name here, it  was already used by xiangshan docker support